TRUEConnector/be-dataapp_resources_consumer/application-docker.properties

```diff
@@ -13,6 +14,7 @@ application.proxyPort=8183
 server.ssl.enabled=${SERVER_SSL_ENABLED}
```

Comment:
Used in application? This is hardcoded in java to always use httpS
Be sure to check and remove from docker-compose.yml and .env file
For both consumer and provider

Reply:
Good thing that you noticed. Removed it